docs/source/datasets/index.rst

```diff
@@ -75,7 +76,46 @@ The following simulant attributes are included in this dataset:
 
 Household Surveys: ACS and CPS
 ------------------------------
+There are two simulated household survey datasets that can be used: the American
+Communities Survey (ACS) and the Current Population Survey (CPS). 
 
+.. list-table:: **Simulant attributes**
+   :header-rows: 1
+
+   * - Attribute Name
+     - Column Name
+   * - Unique simulant ID
+     - 
+   * - Household ID 
+     -  
```

Comment:
missing col name

Reply:
As noted in the doc session, I don't think `household_id` is actually in the household survey data yet -- not sure when the engineers are planning to add it